Improve FW Checkmate (#4672)
endless-sky/endless-sky: data/human/free worlds checkmate.txt

			`As soon as you land, you and Katya contact Tomek and inform him that the system is yours. "Excellent," he says. "Press on with your fleet and take Vega before they have time to regroup. Another Wolf Pack fleet will meet you there for the final assault on the Sol system."`
			choice
				`	"Understood. We will depart immediately."`
					goto end
				`	"Are we really sure we want to attack Earth?"`
				`	"Shouldn't we just be able to stall our time until the Syndicate prove our innocence?"`
					goto syndicate
			`	"Have they given us any other choice?" asks Tomek. "We tried diplomacy. And we cannot hope to defeat the entire Navy.`
				goto orders
			label syndicate

> 			`	"I wish we could just stall here, but we don't have the means to defend ourselves if the Navy decide to mount a counteroffensive large enough to overpower our nukes.`

Reviewer: ```suggestion
			`	"I wish we could just stall here, but we don't have the means to defend ourselves if the Navy decides to mount a counteroffensive large enough to overpower our nukes.`
```

mission "FWC Terrorists 1"
	name "Deal with the terrorists"
	description "Now that the Pug appear to have been driven off, the Syndicate have tracked down the rest of the terrorists. Travel to <destination> to meet with the CEO of the Syndicate."
	source "Pugglemug"
	destination "Foundry"
	passengers 1
	to offer
		has "FWC Pug 6: done"
		not "FWC End: offered"
	on fail
		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
	on offer
		"reputation: Republic" += 20
		conversation
			`You find Admiral Danforth inspecting the Pug spaceport. "This is all very odd," he says. "I'm glad the aliens appear to have left, but I wish we knew where they came from and whether they're likely to return."`
			`	Freya says, "What should we do now?"`

> 			`	"While we were busy fighting the Pug, the Syndicate claimed in front of Parliament that they've tracked down the culrpits of the bombings, proving your innocence," Danforth says. "Your Senate agreed to an end to the war and to the Free Worlds being a semi-autonomous district of the Republic, similar to the Syndicate systems."`

Reviewer: ```suggestion
			`	"While we were busy fighting the Pug, the Syndicate claimed in front of Parliament that they've tracked down the culprits of the bombings, proving your innocence," Danforth says. "Your Senate agreed to an end to the war and to the Free Worlds being a semi-autonomous district of the Republic, similar to the Syndicate systems."`
```

			`	Freya receives a message from Alondo. "Freya, <first>. I've just received word from the Syndicate that they've tracked down where the terrorists are. The CEO of the Syndicate requested that you meet with him on <destination> to discuss how to approach the situation."`

> 			`	"I'll have my ships ready to assist you when you're make your move to assault the terrorists. It's only fitting that you lead the charge against them when you've been the ones blamed for the bombings all this time," Danforth says.`

Reviewer: ```suggestion
			`	"I'll have my ships ready to assist you when you make your move to assault the terrorists. It's only fitting that you lead the charge against them when you've been the ones blamed for the bombings all this time," Danforth says.`
```

			`	"Before the war, we began to work heavily toward developing automation for use on ships. You may have bought and used one of our Vanguards. Such a small crew needed to man a ship of such a size. We made some breakthroughs during the war that allowed us to drastically reduce the crew needed to pilot ships of any size, and we'll be sending a fleet of such ships as cannon fodder to have these terrorists waste their nukes."`
			`	Freya looks shocked, "You're sending people on a suicide mission to take the nukes out of play? Can't we use diplomacy to stop adding bodies to the pile that this war has created?"`
			`	Korban says, "These are terrorists, dear. They aren't to be negotiated with. But fear not, because these ships won't be manned. They'll be fully autonomous warships. Their combat skills are rough around the edges, and this will be an expensive loss - each of these ships costs us at least tens times the production of a normal ship - but much cheaper than the loss of innocent human lives."`
			choice
				`	"So where do we come into play?"`
					goto play
				`	"What are you going to do with this automation technology after this is all said and done?"`

> 			`	"This technology is far from being profitable, but we hope to some day have the skies full of autonomous freighters transporting commodities across the galaxy faster than any merchant ever could, defended by deadly autonomous warships able to outsmart any pirate threat. That's neither here nor there, though. There are more important matters at hand.`

Reviewer: ```suggestion
			`	"This technology is far from being profitable, but we hope to someday have the skies full of autonomous freighters transporting commodities across the galaxy faster than any merchant ever could, defended by deadly autonomous warships able to outsmart any pirate threat. That's neither here nor there, though. There are more important matters at hand.`
```

				goto end
			
			label play
			`	"We've already sent a single autonomous fleet to draw most of the terrorists' fire, but it wasn't enough. We have a second fleet at the ready, though, one for you to command. Being a skilled captain, I'm sure you'll be able to trick the terrorists into wasting whatever remains of their nukes before a proper fleet arrives to deal with them. We've contacted the Navy and told them where the terrorists are located, so they should be arriving shortly after you."`
			
			label end

> 			`	Korban tells you that the second automamous fleet for you to lead is on Hephaestus. "The terrorists are holed up on Stormhold," says Korban, "an old pirate world. We've also made available to you some of our new technology, developed during the war. It may prove helpful to you during this final battle. Good luck, Captain."`

Reviewer: ```suggestion
			`	Korban tells you that the second autonomous fleet for you to lead is on Hephaestus. "The terrorists are holed up on Stormhold," says Korban, "an old pirate world. We've also made available to you some of our new technology, developed during the war. It may prove helpful to you during this final battle. Good luck, Captain."`
```
